### DISCUSSION TITLE
Add an optional preview feature for Markdown and Katex

### DISCUSSION THREAD

**Discussion by 4444dogs** [stoatchat/discussions#398] [Feature Suggestions]
I think having a preview feature would be really nice so you know what your message will look like before you send it. This would work by having a button on the left side of the text box that has a magnifying glass or something.

**Comment by ccrowles** [comment]
This would be great, often I misremember the markdown and fuck up, and on Stoat there's currently no way of knowing if you used the markdown properly until you hit send. If for whatever reason the extra button would be a problem, even just a keyboard shortcut would be great.

**Comment by sudo-nano** [comment]
I'd love this as well.
